### DISCUSSION TITLE
gMKVExtractGUI.v2.6.3更新了，希望跟进汉化

### DISCUSSION THREAD

**Discussion by shuaiyewl** [hooke007/mpv_PlayKit#193] [其它播放器]
gMKVExtractGUI.v2.6.3更新了，希望跟进汉化

**Comment by hooke007** [comment]
我的代办事项上目前除了mpv相关的，其它的优先级都很低。

不会特意跟进每个（小）版本的更新。
如果没有显著新特性的追加或严重漏洞的修复，待办的优先级进一步降低。

**Comment by toothpaste-uploader** [comment]
gMKVExtractGUI.v2.2.2.9 不会用，该怎么才能把.mks 文件提出字幕txt
谢谢大佬，请帮忙指点一下！
